substreams-helper/src/rpc.rs

```diff
@@ -0,0 +1,64 @@
+use substreams::Hex;
+use substreams_ethereum::{pb::eth as ethpb, rpc};
+use tiny_keccak::{Hasher, Keccak};
+
+#[derive(Debug)]
+pub struct RpcCallParams {
+    pub to: Vec<u8>,
```

Comment:
Any particular reason that `to` is a byte array while `method` is a String? I think we should keep them consistent

Reply:
Because bytes are for machine while strings are for human. From a user perspective, it is easiest to just put a string for method name; while for contract address, user never have to manually input, that field, therefore I use bytes, which is more native to substreams

Reply:
should we change const addresses to byte arrays as well then? We should decide how we want to treat addresses and have it be consistent across the code base

Reply:
good point, i end up using this:

```
const CONFIG: Config = Config {
    ethereum: NetworkConfig {
        yearn_lens_oracle: hex!("83d95e0d5f402511db06817aff3f9ea88224b030"),
        usdc_decimals: 6,
    },
};
```

`hex!("83d95e0d5f402511db06817aff3f9ea88224b030")` produces a byte array